experiments/conductor/cmd/runner/runner.go

```diff
@@ -205,6 +210,10 @@ func printHelp() {
 	log.Println("\t3 - Delete the local github branches from the metadata")
 	log.Println("\t4 - Create script.yaml for mock gcp generation in each github branch")
 	log.Println("\t5 - Create _http.log for mock gcp generation in each github branch")
+	log.Println("\t6 - Generate mock Service and Resource go files in each github branch")
+	log.Println("\t7 - Add service to mock_http_roundtrip.go in each github branch")
+	log.Println("\t8 - Add proto to makefile in each github branch")
```

Comment:
Which proto are you referring here? Just some thoughts: maybe we can add all protos ahead to save this step.